README.md

```
@@ -98,6 +101,111 @@ If the documentation doesn't cover what you need, search the [many questions on
 
 If this isn't passing, is there something you can do to help?
 
+## Running Load Tests in AWS
+
+This template supports running load tests on AWS using k6, a modern load testing tool, to evaluate the performance of your application under various conditions. You can automate this process using a custom bash script that provisions an EC2 instance, attaches an IAM role, creates an S3 bucket for storing the results, and executes the k6 load tests.
+
+### Steps for Running AWS Load Tests
+
+#### 1. **Configure AWS CLI**:
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)